src/main/java/com/google/api/codegen/viewmodel/SimpleMessagePropertyView.java

```diff
@@ -25,7 +26,7 @@
  * https://tools.ietf.org/html/draft-zyp-json-schema-03#section-5.7.
  */
 @AutoValue
-public abstract class SimpleMessagePropertyView {
+public abstract class SimpleMessagePropertyView implements ViewModel {
```

Comment:
That makes sense, given what's in the ViewModel interface. I merged the branch already but I'll take that out in the next PR.